stories/core/Header.stories.tsx

```diff
@@ -0,0 +1,209 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Meta, Story } from '@storybook/react/types-6-0';
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+
+import {
+  DropdownMenu,
+  getUserColor,
+  Header,
+  HeaderProps,
+  IconButton,
+  Input,
+  MenuItem,
+  UserIcon,
+} from '../../src/core';
+import HeaderButton from '../../src/core/Header/HeaderButton';
+import HeaderLogo from '../../src/core/Header/HeaderLogo';
+import HeaderBreadcrumbs from '../../src/core/Header/HeaderBreadcrumbs';
+import SvgHelpCircularHollow from '@itwin/itwinui-icons-react/cjs/icons/HelpCircularHollow';
+import SvgVersion from '@itwin/itwinui-icons-react/cjs/icons/Pin';
+import SvgNetwork from '@itwin/itwinui-icons-react/cjs/icons/Network';
+import SvgNotification from '@itwin/itwinui-icons-react/cjs/icons/Notification';
+
+export default {
+  title: 'Core/Header',
+  component: Header,
+  subcomponents: {
+    HeaderBreadcrumbs,
+    HeaderLogo,
+    HeaderButton,
+  },
+  argTypes: {
+    appLogo: { control: { disable: true } },
+    breadcrumbs: { control: { disable: true } },
+    actions: { control: { disable: true } },
+    userIcon: { control: { disable: true } },
+    menuItems: { control: { disable: true } },
+    children: { control: { disable: true } },
```

Comment:
Do the same with `style` and `className`

Reply:
Added